pkg/plugins/manager/process/process.go

```diff
@@ -20,7 +20,7 @@ func ProvideService() *Service {
 }
 
 func (s *Service) Start(ctx context.Context, p *plugins.Plugin) error {
-	if !p.IsManaged() || !p.Backend || p.SignatureError != nil {
+	if !p.IsManaged() || !p.Backend || p.SignatureError != nil || p.Status.Errored {
```

Comment:
This is in theory not needed since the starting step is skipped in the plugin loading pipeline but I added the check just in case.